frontend/package.json

```diff
@@ -45,80 +47,76 @@
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
     "cartodb": "^0.5.1",
-    "clean-webpack-plugin": "^0.1.18",
-    "css-loader": "^0.28.10",
-    "ejs-loader": "^0.3.0",
-    "eslint": "^4.18.1",
+    "clean-webpack-plugin": "^0.1.19",
+    "css-loader": "^0.28.11",
+    "ejs-loader": "^0.3.1",
+    "eslint": "^4.19.1",
     "eslint-config-airbnb": "^16.1.0",
     "eslint-config-prettier": "^2.9.0",
     "eslint-import-resolver-webpack": "^0.8.4",
     "eslint-loader": "^1.9.0",
     "eslint-plugin-import": "^2.9.0",
-    "eslint-plugin-jest": "^21.12.2",
+    "eslint-plugin-jest": "^21.15.0",
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-prettier": "^2.5.0",
-    "eslint-plugin-promise": "^3.6.0",
+    "eslint-plugin-promise": "^3.7.0",
     "eslint-plugin-react": "^7.7.0",
     "eslint-plugin-standard": "^3.0.1",
-    "file-loader": "^1.1.9",
-    "html-webpack-inline-chunk-plugin": "^1.1.1",
-    "html-webpack-plugin": "^2.30.1",
+    "file-loader": "^1.1.11",
+    "html-webpack-plugin": "^3.0.7",
     "husky": "^0.14.3",
-    "image-webpack-loader": "^4.1.0",
-    "jest": "^22.4.2",
+    "image-webpack-loader": "^4.2.0",
+    "jest": "^22.4.3",
     "lint-staged": "^6.1.1",
     "mapshaper": "^0.4.63",
-    "node-sass": "^4.7.2",
-    "postcss-loader": "^2.1.1",
-    "prettier": "^1.10.2",
+    "node-sass": "^4.8.3",
+    "postcss-loader": "^2.1.3",
+    "prettier": "^1.11.1",
+    "redux-logger": "^3.0.6",
     "sass-lint": "^1.12.1",
-    "sass-loader": "^6.0.6",
+    "sass-loader": "^6.0.7",
     "style-loader": "^0.19.1",
-    "uglifyjs-webpack-plugin": "^1.2.2",
     "url-loader": "^0.6.2",
-    "webpack": "^3.11.0",
-    "webpack-dashboard": "^1.1.1",
-    "webpack-dev-server": "^2.11.1",
+    "webpack": "^4.2.0",
+    "webpack-cli": "^2.0.12",
+    "webpack-bundle-analyzer": "^2.11.1",
+    "webpack-bundle-size-analyzer": "^2.7.0",
```

Comment:
whats the difference between `webpack-bundle-analyzer` and `webpack-bundle-size-analyzer`

Reply:
one we need, the other I will remove, because I think it's not needed :P